releases/cloud/point-2025-03-18-release-note.md

```diff
@@ -0,0 +1,38 @@
+---
+description: This article lists improvements and bug fixes in the Syskit Point Cloud version 2025.2.85.20
+---
+
+# March 18, 2025
+
+[Start a free trial](https://www.syskit.com/products/point/free-trial/) and [tell us what you think](https://www.syskit.com/company/contact-us/).
+
+## About Syskit Point Cloud
+
+* **New to Syskit Point Cloud?** Explore Syskit Point Cloud with a 21-day [free trial](https://www.syskit.com/products/point/free-trial/) for an easy and effective way to manage and secure your environment.
+
+* **Already using Syskit Point Cloud?** Syskit Point Cloud is automatically upgraded to the latest version when available. The automatic update occurs outside working hours to ensure minimal interference with your day-to-day business. The new version will start rolling out with this announcement and is expected to reach all customers in the upcoming days.
+
+
+## Improvements & Bug Fixes 
+
+* **The Power Platform** People Picker supports SharePoint lists.
+  * When assigning owners in Power Platform Flows, you can search for and select SharePoint lists used in your connections. 
+
+* **The People Picker now resets** after a user is selected. 
+  * After picking a user, the input field clears automatically, making it easier to search for and select multiple users. 
+
+* **Review Reminders** are now sent automatically to users for any pending reviews, with the option for admins to manually send a reminders when needed. 
```

Comment:
ovo je vezano za Workspace review - novu vrstu reviewa
to je još in development, nije vidljivo pa maknuti